recipes/inforna/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+make clean
+make all VIENNA=$PREFIX
+
+mv INFO-RNA-2.1.2 $PREFIX/bin
```

Comment:
Is it worth adding a prefix not containing the version number?:
`ln -s INFO-RNA-2.1.2 "$PREFIX/bin/INFO-RNA"`